data/i18n/en.yaml

```diff
@@ -236,7 +236,7 @@ social:
   humanstext: Join the LBRY community.
   robotsheader: Robots
   robotstext: Make with us. All LBRY code is open source.
-  slack: Slack (chat)
+  discord: Discord (chat)
```

Comment:
The left side of these declarations is the input to an i18n call, e.g. `__('social.slack'`)`. I think in this case this one isn't used, but generally you can't change the key declaration without also changing the i18n calls themselves.

Reply:
whether this can be changed. instead of now migrating to discord. sir